src/main/java/service/impl/FileParserImpl.java

```diff
@@ -0,0 +1,37 @@
+package service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransactionModel;
+import model.InFileStructure;
+import service.FileParser;
+
+/*
+The main idea of this class is to parse the data from the file
+and to create new instances of FruitTransactionModel class.
+
+- receive String textBox from ReadFromCsvFileImpl
+- return List<FruitTransactionModel>
+ */
+public class FileParserImpl implements FileParser {
+    @Override
+    public List<FruitTransactionModel> getFruitTransaction(String data) {
+        InFileStructure inFileStructure;
```

Comment:
useless variable
```suggestion
```

Reply:
To understand the structure of the file, it will be enough to look at the file, it is not necessary to create classes for this